modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -861,11 +1280,6 @@ static void run_arithm_s(Buffer &dst, const View &src, const float scalar[4], Ar
 
     switch (arithm)
     {
-    case ARITHM_ABSDIFF:
-        for (int w=0; w < width; w++)
-            for (int c=0; c < chan; c++)
-                out[chan*w + c] = absdiff<DST>(in[chan*w + c], scalar[c]);
-        break;
```

Comment:
It's necessary to avoid unnecessary template specializations for combination of types which handles by other arithm kernels such as Add, Sub, Mul and Div.

Reply:
For example type combination when SRC = uchar DST=float. There is no this type combination in AbsDiffC kernel.

Reply:
OK, it makes sense, I agree, AbsDiff should actually be separated from other arithmetic operations. Thanks!